### PR TITLE
Refactored appstore

### PR DESCRIPTION
**Pull request by SamD97**
Issue: Common in properties in `&.appstore` and `&.playstore`

Solution: Common properties were moved out to the parent.

### CODE REVIEW

**Comment by shahednasser** [comment]
Regarding the comment you mentioned I didn't notice the different left attributes, so put those with the content instead of in the parent element.

**Comment by SamD97** [comment]
> Regarding the comment you mentioned I didn't notice the different left attributes, so put those with the content instead of in the parent element.

I am slightly confused, you meant to say "I did notice the different `left` attributes" right?
https://github.com/sButtons/sbuttons/blob/82dfc98c10f6020f1797b537b5cba436abd7333b/src/components/icon/_appstore.less#L46
https://github.com/sButtons/sbuttons/blob/82dfc98c10f6020f1797b537b5cba436abd7333b/src/components/icon/_appstore.less#L69

Anyways I am assuming that's what you meant based on the latter part of your reply and undo that change.

**Review by shahednasser — CHANGES_REQUESTED** [review]
So I checked the current website and apparently the different left attributes (100px and 87px) is only applied when the button is block, else it's always 50px so can we refactor the code to portray that? Change the 87px to 102px as a fix to the original button's design.

Also for the `.appstore-btn` when it's a block button please change the padding left to 30px as a fix to the original button.

File: dist/sbuttons.css [hunk 1]
```diff
@@ -1924,10 +1924,12 @@ a.yellow-btn:active:not(.fill-color-btn) {
 }
 .appstore-btn {
   box-shadow: 0px 1px 4px #4d4d4d;
+  background-color: #000000;
+  color: #fff;
   transition: all 0.2s;
   position: relative;
   padding-top: 15px;
-  padding-left: 60px;
+  padding-left: 30px;
```
Comment (SamD97) [inline dist/sbuttons.css:1932]: The `padding-left: 30px` only affects `.block-btn` because of line 31

File: src/components/icon/_appstore.less [hunk 2]
```diff
@@ -1,9 +1,11 @@
 .appstore-btn {
   box-shadow: 0px 1px 4px #4d4d4d;
+  background-color: #000000;
+  color: #fff;
   transition: all 0.2s;
   position: relative;
   padding-top: 15px;
-  padding-left: 60px;
+  padding-left: 30px;
```
Comment (shahednasser) [inline src/components/icon/_appstore.less:8]: This change should only be applied to `applestore.block-btn` not all. Please fix this and change the default back to 60px

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: src/components/icon/_appstore.less (hunk 2 above)
Comment (SamD97) [inline src/components/icon/_appstore.less:8]: The `padding-left: 30px` only affects `.block-btn` because of line 31 hence for other buttons it is `50px`

**Comment by shahednasser** [comment]
Great work thank you!

**Comment by SamD97** [comment]
Thank you for all the help and feedback :+1:

**Pull request merged**